scene/2d/tile_map.h

```diff
@@ -50,7 +50,9 @@ class TileMap : public Node2D {
 
 	enum HalfOffset {
 		HALF_OFFSET_X,
+		HALF_OFFSET_NEGATIVE_X,
```

Comment:
this change breaks compatibility, when loading an old tilemap it will read "1" values as HALF_OFFSET_NEGATIVE_X instead of HALF_OFFSET_Y, and the difference will propagate

Reply:
Would you recommend we move the new enum members to the bottom or assign explicit values that keep compatibility? (I'm working on this with @Byteron )

Reply:
totally recomended